transport_nantes/mailing_list/views.py

```diff
@@ -340,7 +340,7 @@ def get_context_data(self, **kwargs) -> dict:
             raise Http404()
 
         try:
-            send_record = TopicBlogEmailSendRecord.objects.get(
+            send_record = SendRecordMarketingEmail.objects.get(
```

Comment:
Transactional

Reply:
you unsubscribe from a mailing list. Sending mails to mailing list is saved as a SendRecordMarketing because it is Marketing right ?
So when one clicks on unsub, the page loads a page "Do you want to unsub from X mailing list ?" where X is extracted from the original send record. Which is Marketing. 
How is it transactional ?